lib/config.js

```diff
@@ -6,15 +6,15 @@ import {config} from 'bedrock-web';
 // set bedrock-web-wallet default config
 config.wallet = {
   defaults: {
-    edvBaseUrl: `https://localhost:55443/edvs`,
+    edvBaseUrl: 'https://localhost:55443/edvs',
     // FIXME: do not use `referenceId` and put that information on the server,
     // rather have profile agent information store a location to read EDV IDs
     // from based on local names; remove this from the shared web app config
     edvs: {
-      users: _getReferenceId('users'),
-      credentials: _getReferenceId('credentials')
+      users: encodeURIComponent('https://localhost:38443/users'),
```

Comment:
This reference id does not match what the previous function outputs. Please fix both reference ids for the edvs.

Reply:
@gannan08 i was told on irc that those should be changed to those urls. Should this be encodeURIComponent('localhost:users') and encodeURIComponent('credentials')  instead?